src/cctag/utils/Exceptions.hpp

```diff
@@ -73,6 +73,8 @@ class error_info<Tag, error_info_sstream>: public exception_detail::error_info_b
 		_value._v << value;
 	}
 
+    	error_info_base * clone() const {}
```

Comment:
Are we not supposed to do something like that?
```
error_info_base * clone() const
{
  error_info* p = new error_info();
  *p = *this;
  return p;
}
```
To create a new instance with the same content.

Reply:
Yes, absolutely. That looks correct.